torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1005,6 +1005,35 @@ def sample_inputs(self, device, dtype, requires_grad=False):
         return out
 
 
+def sample_inputs_householder_product(op_info, device, dtype, requires_grad):
+    """
+    This function generates input for torch.householder_product (torch.orgqr).
+    The first argument should be a square matrix or batch of square matrices, the second argument is a vector or batch of vectors.
+    Empty, square, rectangular, batched square and batched rectangular input is generated.
+    """
+    samples = (
+        SampleInput((make_tensor((0, 0), device, dtype, low=None, high=None, requires_grad=requires_grad),
+                    make_tensor((0,), device, dtype, low=None, high=None, requires_grad=requires_grad))),
+
+        SampleInput((make_tensor((S, S), device, dtype, low=-2, high=2, requires_grad=requires_grad),
```

Comment:
Would you elaborate on the range here?

Reply:
Each column of the matrix is getting multiplied many times leading to very large values for the Jacobian matrix entries and making the finite-difference result of grad check less accurate. That's why gradcheck with the default range [-9, 9] fails.

Reply:
Cool; would you add a comment explaining that?